frontend/src/pages/docs/[...slug].tsx

```diff
@@ -403,7 +403,7 @@ DocPage.Layout = function DocLayout({
   return (
     <Layout>
       <Head>
-        <title>cellxgene | documentation</title>
+        <title>CELLxGENE | documentation</title>
```

Comment:
```suggestion
        <title>CELLxGENE | Documentation</title>
```

Sneaking in a change. :)